cves/kernel/CVE-2016-9685.yml

```diff
@@ -235,9 +246,11 @@ interesting_commits:
       * Other commits that fixed a similar issue as this vulnerability
       * Anything else you find interesting.
   commits:
-  - commit:
-    note:
-  - commit:
+  - commit: 726801ba067410a1d38518823f2c253a087f6c6f
```

Comment:
Can you update the vcc up above then?

Reply:
Good callout! I wasn't sure if I was supposed to do that